tools/pepsi/src/cargo.rs

```diff
@@ -17,8 +18,11 @@ pub struct Arguments {
     /// Pass through arguments to cargo ... -- PASSTHROUGH_ARGUMENTS
     #[arg(last = true, value_parser)]
     pub passthrough_arguments: Vec<String>,
+    #[arg(long)]
```

Comment:
Please add documentation comments like for the other flags.

Reply:
Done